src/sentry/integrations/aws_lambda/integration.py

```diff
@@ -102,6 +102,12 @@ def serialize_lambda_function(self, function):
             latest_version = get_latest_layer_version(function)
             current_version = get_version_of_arn(sentry_layer)
             out_of_date = latest_version > current_version
+
+            # If env variable "SENTRY_INITIAL_HANDLER" is not present, then
+            # it is should be assumed that this function is not enabled!
+            env_variables = function.get("Environment", {}).get("Variables", {})
+            if "SENTRY_INITIAL_HANDLER" not in env_variables:
```

Comment:
@ahmedetefy we should only be checking this for the Python runtimes since it's not relevant for Node.

Reply:
Good catch! Thanks, Fixed!